Specs/Petstore/generated/Swift/Sources/APIClient.swift

```diff
@@ -123,24 +123,8 @@ public class APIClient {
                         }
                     }
                 },
-                with: urlRequest,
-                encodingCompletion: { result in
-                    switch result {
-                    case .success(let uploadRequest, _, _):
-                        cancellableRequest.networkRequest = uploadRequest
```

Comment:
Is all this still required?

Reply:
@yonaskolb Thank you for your comment! Sorry, I'm not sure your comment so could you describe more detail?

Reply:
Alamofire 5 removes `encodingCompletion`.
https://stackoverflow.com/questions/54985001/alamofire-5-upload-encodingcompletion

Reply:
@yonaskolb  Oh, I got it. I have to add `responseData` method to handle response.
I will revise this code to as follows. Did I get that right? 
```
cancellableRequest.networkRequest = sessionManager.upload(
    multipartFormData: { multipartFormData in
        for (name, value) in request.formParameters {
            if let file = value as? UploadFile {
                switch file.type {
                case let .url(url):
                    if let fileName = file.fileName, let mimeType = file.mimeType {
                        multipartFormData.append(url, withName: name, fileName: fileName, mimeType: mimeType)
                    } else {
                        multipartFormData.append(url, withName: name)
                    }
                case let .data(data):
                    if let fileName = file.fileName, let mimeType = file.mimeType {
                        multipartFormData.append(data, withName: name, fileName: fileName, mimeType: mimeType)
                    } else {
                        multipartFormData.append(data, withName: name)
                    }
                }
            } else if let url = value as? URL {
                multipartFormData.append(url, withName: name)
            } else if let data = value as? Data {
                multipartFormData.append(data, withName: name)
            } else if let string = value as? String {
                multipartFormData.append(Data(string.utf8), withName: name)
            }
        }
    },
    with: urlRequest
).responseData(queue: decodingQueue) { dataResponse in
    self.handleResponse(request: request, requestBehaviour: requestBehaviour, dataResponse: dataResponse, completionQueue: completionQueue, complete: complete)
}
```